webapp/components/User/User.vue

```diff
@@ -79,7 +79,7 @@
             />
           </ds-flex-item>
           <ds-flex-item :width="{ base: 1 }">
-            <ds-button fullwidth>
+            <ds-button fullwidth @click="block(user)">
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-11-04T19:47:02Z" title="Monday, November 4th 2019, 8:47:02 pm +01:00">Nov 4, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -79,7 +79,7 @@
             />
           </ds-flex-item>
           <ds-flex-item :width="{ base: 1 }">
-            <ds-button fullwidth>
+            <ds-button fullwidth @click="block(user)">
```


Honestly, I would like to get this button completely removed. It's enough to have it on the user profile page.

This button exists since the early days of our nitro code base. It never worked and this has been discovered just a few days ago. Apparently not too many users expect the block button in the popup.

Reply:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-11-05T06:43:02Z" title="Tuesday, November 5th 2019, 7:43:02 am +01:00">Nov 5, 2019</time>_

---


Oh, didn't know that! I'm fine with removing it – @ogerly what do you think?

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2019-11-05T13:04:14Z" title="Tuesday, November 5th 2019, 2:04:14 pm +01:00">Nov 5, 2019</time>_

---


dennis has asked me to do this. Please discuss this with dennis.